Makefile

```diff
@@ -369,6 +369,15 @@ dist: ## Generates binaries for a Mimir release.
 		done; \
 		touch $@
 
+build-mixin: check-mixin-jb
+	@rm -rf $(MIXIN_OUT_PATH) && mkdir $(MIXIN_OUT_PATH)
+	@mixtool generate all --output-alerts $(MIXIN_OUT_PATH)/alerts.yaml --output-rules $(MIXIN_OUT_PATH)/rules.yaml --directory $(MIXIN_OUT_PATH)/dashboards ${MIXIN_PATH}/mixin-compiled.libsonnet
```

Comment:
We should update mixtool to latest master. When people run it locally it doesn't work anymore. (But that's for separate PR)

Reply:
Ok, will do in a separate PR.

Reply:
I started the work here, but don't have the time to continue it right now:
https://github.com/grafana/mimir/compare/update-mixtool

_Leaving it for the future_

Reply:
Opened an issue about it:
https://github.com/grafana/mimir/issues/1617

Reply:
Thanks.